java/compaction/compaction-job-creation/src/test/java/sleeper/compaction/job/CompactionJobTest.java

```diff
@@ -21,221 +21,19 @@
-        boolean equals1 = compactionJob1.equals(compactionJob2);
-        boolean equals2 = compactionJob1.equals(compactionJob3);
-        boolean equals3 = compactionJob1.equals(compactionJob4);
-        int hashCode1 = compactionJob1.hashCode();
-        int hashCode2 = compactionJob2.hashCode();
-        int hashCode3 = compactionJob3.hashCode();
-        int hashCode4 = compactionJob4.hashCode();
-
-        // Then
-        assertThat(equals1).isTrue();
-        assertThat(equals2).isFalse();
-        assertThat(equals3).isFalse();
-        assertThat(hashCode2).isEqualTo(hashCode1);
-        assertThat(hashCode3).isNotEqualTo(hashCode1);
-        assertThat(hashCode4).isNotEqualTo(hashCode1);
-    }
-
-    @Test
-    public void testEqualsAndHashCodeForSplittingJobWithIterator() {
-        // Given
-        CompactionJob compactionJob1 = CompactionJob.builder()
-                .tableName("table")
-                .jobId("job-1")
-                .inputFiles(Arrays.asList("file1", "file2", "file3"))
-                .outputFile("outputFile")
-                .isSplittingJob(true)
-                .partitionId("partition1")
-                .childPartitions(Arrays.asList("childPartition1", "childPartition2"))
-                .dimension(2)
-                .iteratorClassName("Iterator.class")
-                .iteratorConfig("config1").build();
-        CompactionJob compactionJob2 = CompactionJob.builder()
-                .tableName("table")
-                .jobId("job-1")
-                .inputFiles(Arrays.asList("file1", "file2", "file3"))
-                .outputFile("outputFile")
-                .isSplittingJob(true)
-                .partitionId("partition1")
-                .childPartitions(Arrays.asList("childPartition1", "childPartition2"))
-                .dimension(2)
-                .iteratorClassName("Iterator.class")
-                .iteratorConfig("config1").build();
-        CompactionJob compactionJob3 = CompactionJob.builder()
-                .tableName("table")
-                .jobId("job-1")
-                .inputFiles(Arrays.asList("file1", "file2", "file3"))
-                .outputFile("outputFile1")
-                .isSplittingJob(true)
-                .partitionId("partition1")
-                .childPartitions(Arrays.asList("childPartition1", "childPartition2"))
-                .dimension(2)
-                .iteratorClassName("Iterator2.class")
-                .iteratorConfig("config1").build();
-
-        // When
-        boolean equals1 = compactionJob1.equals(compactionJob2);
-        boolean equals2 = compactionJob1.equals(compactionJob3);
-        int hashCode1 = compactionJob1.hashCode();
-        int hashCode2 = compactionJob2.hashCode();
-        int hashCode3 = compactionJob3.hashCode();
-
-        // Then
-        assertThat(equals1).isTrue();
-        assertThat(equals2).isFalse();
-        assertThat(hashCode2).isEqualTo(hashCode1);
-        assertThat(hashCode3).isNotEqualTo(hashCode1);
```

Comment:
Why were the above tests deleted?

Reply:
I think we couldn't see a need for them. I can bring them back though.